Gemfile

```
@@ -4,9 +4,8 @@ source 'https://rubygems.org'
 gemspec
 
 # Private runtime dependencies
-gem 'nokogiri', '~> 1.13', force_ruby_platform: true
-gem 'sisjwt', '~> 0.2.4',
-    github: 'tractionguest/sisjwt-ruby'
+gem 'nokogiri', '~> 1.13', platforms: [:ruby]
+gem 'sisjwt', github: 'tractionguest/sisjwt-ruby', ref: '627b919'
```

change to
```suggestion
gem 'sisjwt', '~> 0.4.0', github: 'tractionguest/sisjwt-ruby'
```
when this [PR](https://github.com/tractionguest/sisjwt-ruby/pull/7) is merged